### PR TITLE
feat: Adding Paidy as New APM

### PR DESCRIPTION
**Pull request by sathyashyam2406**
JIRA: https://engineering.paypalcorp.com/jira/browse/DTALTPBLR-756

### CODE REVIEW

File: dist/module/funding.js [hunk 1]
```diff
@@ -55,4 +56,4 @@ export var FUNDING_PRODUCTS = {
   PAY_IN_4: "payIn4",
   PAYLATER: "paylater",
   CREDIT: "credit"
-};
\ No newline at end of file
+};
```
Comment (gregjopa) [inline dist/module/funding.js:59]: Please remove the "dist" folder changes in this PR. Our publishing process takes care of updating the dist folder.

**Comment by gregjopa** [comment]
Hi @sathyashyam2406 please rebase with the main branch to resolve the build failure. Thank you!

File: dist/module/funding.js (hunk 1 above)
Comment (sathyashyam2406) [reply dist/module/funding.js]: Removed the "dist" folder

**Comment by sathyashyam2406** [comment]
> Hi @sathyashyam2406 please rebase with the main branch to resolve the build failure. Thank you!

Have rebased with the main branch to fetch latest changes

**Review by wsbrunson — APPROVED** [review]

**Pull request merged**